core/src/hir/attrs.rs

```diff
@@ -636,51 +577,22 @@ pub struct BackendAttrSupport {
     pub utf8_strings: bool,
     /// Whether the language uses UTF-16 strings
     pub utf16_strings: bool,
-
-    // Special methods
-    /// Marking a method as a constructor to generate special constructor methods.
-    pub constructors: bool,
-    /// Marking a method as a named constructor to generate special named constructor methods.
-    pub named_constructors: bool,
     /// Marking constructors as being able to return errors. This is possible in languages where
     /// errors are thrown as exceptions (Dart), but not for example in C++, where errors are
     /// returned as values (constructors usually have to return the type itself).
     pub fallible_constructors: bool,
-    /// Marking methods as field getters and setters, see [`SpecialMethod::Getter`] and [`SpecialMethod::Setter`]
-    pub accessors: bool,
```

Comment:
Okay, actually, come to think of it, we _don't_ want to remove the ability to cfg on this: we just want the cfg to be unnecessary for the default.

I think it's necessary to still be able to cfg on these because you want the ability to say things like "if this does _not_ support comparators, disable". This was deliberate design.